service/cloudconfig/cloud_config.go

```diff
@@ -0,0 +1,33 @@
+package cloudconfig
+
+import (
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig"
+)
+
+type MasterExtension struct{}
+
+func (me *MasterExtension) Files() ([]k8scloudconfig.FileAsset, error) {
```

Comment:
Files is empty because the certificates will be stored in Key Vault.

Reply:
Those are very good comment. I wonder why you don't make them godoc.

Reply:
Comments added :)